src/Compilers/VisualBasic/Portable/Scanner/CharacterInfo.vb

```diff
@@ -488,4 +488,37 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
+    Friend Module CharExts
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsAnyOf(c As Char, c0 As Char, c1 As Char) As Boolean
```

Comment:
Unfortunately, XML documentation doesn't help at a call site unless you are reading the code in VS.